app/vep/models/pipeline_model.py

```diff
@@ -1,55 +1,69 @@
 from typing import List
-from pydantic import BaseModel
-from core.config import NF_COMPUTE_ENV_ID, UPLOAD_DIRECTORY
+from pydantic import BaseModel, model_serializer
+from core.config import (
+    NF_COMPUTE_ENV_ID,
+    UPLOAD_DIRECTORY,
+    NF_PIPELINE_URL,
+    NF_WORK_DIR,
+)
 from core.logging import InterceptHandler
-import logging
+import logging, json, os
 
 logging.getLogger().handlers = [InterceptHandler()]
 
-import tempfile 
 
 class VEPConfigParams(BaseModel):
-  vcf: str
-  vep_config: str
+    vcf: str
+    vep_config: str
+
+    @model_serializer
+    def vep_config_serialiser(self):
+        vcf_str = '"vcf":"' + self.vcf + '"'
+        config_str = '"vep_config":"' + self.vep_config + '"'
+        stringified_encoded_json = "{" + vcf_str + "," + config_str + "}"
```

Comment:
Is this the preferred way of concatenating/interpolating/templating strings? You don't prefer f-strings?

Reply:
With formatting the variable should be enclosed in { } but I need it in the string itself . Also, the way the Pedantic serialiser works this is the only thing which is working I am afraid. I will try to play around with formatting option and see if it works.